README.md

```diff
@@ -170,6 +173,60 @@ $ grunt bump:git
 >> Committed as "Release v1.0.1-ge96c"
 >> Tagged as "v1.0.1-ge96c"
 >> Pushed to origin
+
+$ grunt bump:prepatch
+>> Version bumped to 1.0.2-rc.0
+>> Committed as "Release v1.0.2-rc.0"
+>> Tagged as "v1.0.2-rc.0"
+>> Pushed to origin
+
+$ grunt bump:prerelease
```

Comment:
bump:prerelease is how you bump the prerelease version.
From a released version (1.0.0) you choose
* prepatch  -> 1.0.1-rc.0     prerelease -> 1.0.1-rc.1
* preminor  -> 1.1.0-rc.0     prerelease -> 1.1.0-rc.1
* premajor  -> 2.0.0-rc.0     prerelease -> 2.0.0-rc.1


The way it is now (and in the readme) is incorrect.  
* Current behavior. v1.0.0 bump:prerelease -> v1.0.0-1
1.0.0-1 should be the prerelease for 1.0.0 so this should not be the behavior.

re the defaulted prreleaeName.  There needs to be something other than an empty string as the prereleaseName otherwise the the versions are not bumped properly.  They behave more likethe existing functionality of prerelease which is incorrect.